src/data/set/pointwise.lean

```diff
@@ -54,6 +54,17 @@ set multiplication, set addition, pointwise addition, pointwise multiplication,
 pointwise subtraction
 -/
 
+/--
+Pointwise monoids (`set`, `finset`, `filter`) have derived pointwise actions of the form
+`has_scalar α β → has_scalar α (set β)`. When `α` is `ℕ` or `ℤ`, this action conflicts with the
+nat or int action coming from `set β` being a `monoid` or `div_inv_monoid`. For example,
+`2 • {a, b}` can both be `{2 • a, 2 • b}` and `{a + a, a + b, b + a, b + b}`.
```

Comment:
I think it would valuable to refer to both instances by name here, `set.has_scalar` and `set.has_nsmul`.